packages/workflow/docs/WorkflowExample.stories.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react"
+import { Meta, StoryFn } from "@storybook/react"
+// import { Button } from "@kaizen/button"
+// import VisibleIcon from "@kaizen/component-library/icons/visible.icon.svg"
+import { Button } from "@kaizen/button"
+import { ComponentDocsTemplate } from "../../../storybook/components/DocsContainer"
+import { Workflow, WorkflowExit, Footer, Header, WorkflowProps } from "../"
+
+const defaultArgs = {
+  workflowName: "Create a self-reflection cycle",
+  stepName: "Settings",
+  steps: ["Settings", "Questions", "Preview", "Employees", "Schedule"],
+  isComplete: false,
+  status: {
+    content: "Draft",
+    vairant: "statusDraft",
+  },
+  previousAction: <Button reversed label="back" />,
+  nextAction: <Button reversed label="next" />,
+  actions: [
+    <WorkflowExit
+      key={"would-use-uui"}
+      exitLabel="Save and close"
+      exitTitle="Before you exit"
+      exitDescription="Your content has not yet been saved. Click the button below or discard the changes"
+      confirmExitLabel="Close and save"
+      dismissExitLabel="Dismiss"
+      onExit={(): void => alert("mock example of a save action")}
+    />,
+  ],
+} satisfies WorkflowProps
+
+const meta = {
+  tags: ["autodocs"],
+  title: "Workflow/Custom Workflow",
+  component: Workflow,
+  parameters: {
+    docs: {
+      source: { type: "code" },
+      container: ComponentDocsTemplate,
+    },
+    installation: [
+      "npm install @kaizen/workflow",
+      "import { Header, Footer } from `@kaizen/workflow`",
+    ],
+    resourceLinks: {
+      sourceCode:
+        "https://github.com/cultureamp/kaizen-design-system/tree/main/packages/workflow/",
+      figma:
+        "https://www.figma.com/file/IJTy1JpS4Xyop5cQwroRje/%F0%9F%9B%A0%EF%B8%8F-Self-reflection%3A-Build-Handoff?node-id=188%3A62005&t=x4zyx07E2G3BmKGw-1",
+      /** @todo (optional): Add Confluence link */
+      designGuidelines:
+        "https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3064989884/Documentation",
+    },
+  },
+} satisfies Meta<typeof Workflow>
+
+export default meta
+
+/**
+<p>This is a custom version of the `Workflow` assembled using the `Header` and `Footer` exported from `@kaizen/workflow`.</p>
+ */
+export const Playground: StoryFn<WorkflowProps> = ({
```

Comment:
Probably want to use `mdx` for this, cause you don't necessarily need an args table/controls for an example of composition. A simple story with the source code open would suffice I think